requirements-optional.txt

```diff
@@ -1 +1,2 @@
 Mako==1.0.3
+raven
```

Comment:
Would it make sense to limit the range of versions for the package, as applicable for Django 1.8?